crowdsec-docs/docs/appsec/intro.md

```diff
@@ -23,10 +34,55 @@ This component capitalizes on existing remediation functions in web servers (suc
 
 ![appsec-global](/img/appsec-global.svg)
 
+### How it works
```

Comment:
Removed links to pages that may cause churn in the first 30% of the page, still linked at bottom of page.